tools/code-owners-parser/CodeOwnersManualTester/CodeOwnerUtils.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Sdk.Tools.CodeOwnersParser;
+
+namespace CodeOwnersManualTester
+{
+    internal class CodeOwnerUtils
```

Comment:
I see inconsistency in naming:
- `CodeOwner`
- `CodeOwners`

I would suggest to always use `Codeowners`, as this is the convention I adopted in this codebase.

Reply:
Sure. I'll update with the next commit.